docker-compose.yml

```diff
@@ -1,26 +1,134 @@
-version: "3"
+version: '2.4'
 services:
-  redis:
-    image: "redis:4"
-    command: ["redis-server", "--appendonly", "yes"]
-    hostname: redis
-    restart: on-failure
-    volumes:
-      - ~/bridge_data/redis:/data
   rabbit:
-    image: "rabbitmq:3-management"
+    cpus: 0.3
+    mem_limit: 500m
+    environment: ['RABBITMQ_NODENAME=node@rabbit']
     hostname: rabbit
-    environment:
-      - RABBITMQ_NODENAME=node@rabbit
-    volumes:
-      - ~/bridge_data/rabbitmq:/var/lib/rabbitmq/mnesia
-    restart: on-failure
+    image: rabbitmq:3
+    networks:
+      - net_rabbit_bridge_request
+      - net_rabbit_bridge_collected
+      - net_rabbit_bridge_affirmation
+      - net_rabbit_bridge_senderhome
+      - net_rabbit_bridge_senderforeign
+    restart: unless-stopped
+    volumes: ['~/bridge_data/rabbitmq:/var/lib/rabbitmq/mnesia']
+  redis:
+    cpus: 0.1
+    mem_limit: 500m
+    command: [redis-server, --appendonly, 'yes']
+    hostname: redis
+    image: redis:4
+    networks:    
+      - net_db_bridge_request
+      - net_db_bridge_collected
+      - net_db_bridge_affirmation
+      - net_db_bridge_senderhome
+      - net_db_bridge_senderforeign
+    restart: unless-stopped
+    volumes: ['~/bridge_data/redis:/data']
   bridge:
+    cpus: 0.1
+    mem_limit: 500m
+    build: .
+    command: 'true'
+    env_file: ./.env
+    environment: 
+      - NODE_ENV=production 
+    restart: on-failure
+  bridge_request:
+    cpus: 0.1
+    mem_limit: 500m
+    build: .
+    command: 'true'
+    env_file: ./.env
+    environment: 
+      - NODE_ENV=production 
+      - VALIDATOR_ADDRESS=${VALIDATOR_ADDRESS}
```

Comment:
I think we don't need `VALIDATOR_ADDRESS` here since the address will be generated automatically based on the private key provided in the next line.
@patitonar please confirm.

Reply:
That's correct. `VALIDATOR_ADDRESS` is generated automatically using the private key.

Reply:
Hm. I might be mistaking, but it won't work if not specifying the `VALIDATOR_ADDRESS`. However, let me check it first.